pkg/server/server.go

```diff
@@ -224,6 +233,9 @@ func (h *handler) authenticateEndpoint(w http.ResponseWriter, req *http.Request)
 	} else if userMapping, exists := h.lowercaseUserMap[arnLower]; exists {
 		username = userMapping.Username
 		groups = userMapping.Groups
+	} else if _, exists := h.accountMap[identity.AccountID]; exists {
+		groups = []string{}
+		username = identity.Name
```

Comment:
I think this proposal is to map the ARN to the username (with some normalization to restrict to allowed characters if needed).  Otherwise a role and user with the same name would potentially have the same permissions based on RBAC.

Reply:
You mean, username should be something like `role:Alice` or `user:Alice`?

Reply:
I was thinking the full ARN of the user/role.  `arn:aws:iam::012345678901:user/Alice`.  Otherwise an automapped user `Alice` in account `555555555555` would have the same access as `Alice`.